package.json

```diff
@@ -3,46 +3,28 @@
   "version": "1.0.0",
```

Comment:
Version number was fixed in one of the bridges, before being merged to master. An incorrect version number was used in master previously. But that has been corrected now.